apps/test-low-memory.sh

```diff
@@ -17,7 +17,8 @@ STR
   CODECOV=/tmp/codecov.sh
   curl -s https://codecov.io/bash > ${CODECOV}
   chmod +x ${CODECOV}
-  LOG=">"
+  CODECOV="$CODECOV -C $DRONE_COMMIT_SHA"
+  LOG='&& :' # stub
```

Comment:
This change will stub out logging to files in both cases of this conditional. If this is intentional, should we just remove the log-to-file lines entirely and simplify this logic? (e.g., `${LOG} log/unitTest.log`, etc will now always be a no-op in both cases)

Reply:
I think I want to keep it for now and decide after trying the change to run apps tests in parallel on drone. I'm not sure if the resulting output will still be readable without redirecting to files. I'll look into simplifying this script after trying that change either way, though.